src/theory/bv/theory_bv.h

```diff
@@ -180,6 +181,9 @@ class TheoryBV : public Theory {
   /** Index of the next literal to propagate */
   context::CDO<unsigned> d_literalsToPropagateIndex;
 
+  /** Extended theory module, for context-dependent simplification. */
+  std::unique_ptr<ExtTheory> d_extTheory;
```

Comment:
This is minor but since we are always creating it together with `TheoryBv`, does it have to be a `unique_ptr` or could it just be a regular data member?

Reply:
I made it a pointer here since I think it *should* be configurable whether we create it or not :)

Reply:
Ok, that's fair :)